requirements/edx/base.txt

```diff
@@ -91,6 +91,7 @@ django-ratelimit-backend==0.6
 unicodecsv==0.9.4
 django-require==1.0.6
 pyuca==1.1
+wrapt==1.10.5
```

Comment:
Do we need to add this new dependency? Or is there something we've already installed that might suffice?

Also, I don't know how the license terms will work - the package doesn't use a standard license. Have you checked with the open source team about this? https://github.com/GrahamDumpleton/wrapt/blob/develop/LICENSE